src/Conversions_TEST.cc

```diff
@@ -218,6 +218,7 @@ TEST(Conversions, Material)
   workflow.SetEmissiveMap("emissive_map.png");
   workflow.SetGlossinessMap("dummy_glossiness_map.png");
   workflow.SetSpecularMap("dummy_specular_map.png");
+  workflow.SetLightMap("light_map.png", 1u);
```

Comment:
Is the `light_map.png` file suppose to exist?

Reply:
No, I think this is just testing that the conversion between SDF and messages works fine, without worrying about resolving the file.